scylla-cdc-replicator/src/main.rs

```diff
@@ -1,29 +1,65 @@
 mod replication_tests;
 pub mod replicator_consumer;
 
-use replicator_consumer::ReplicatorConsumerFactory;
-use scylla::SessionBuilder;
-use std::error::Error;
-use std::sync::Arc;
 use std::time::Duration;
 
+use clap::Parser;
+
+use replicator_consumer::Replicator;
+
+#[derive(Parser)]
+struct Args {
+    /// Keyspace name
+    #[clap(short, long)]
+    keyspace: String,
+
+    /// Table names provided as a comma delimited string
+    #[clap(short, long)]
+    table: String,
+
+    /// Address of a node in source cluster
+    #[clap(short, long)]
+    source: String,
+
+    /// Address of a node in destination cluster
+    #[clap(short, long)]
+    destination: String,
+
+    /// Window size in seconds
+    #[clap(long, default_value_t = 60.)]
+    window_size: f64,
+
+    /// Safety interval in seconds
+    #[clap(long, default_value_t = 30.)]
+    safety_interval: f64,
+
+    /// Sleep interval in seconds
+    #[clap(long, default_value_t = 10.)]
+    sleep_interval: f64,
+}
+
 #[tokio::main]
-async fn main() -> Result<(), Box<dyn Error>> {
-    println!("Hello");
-    tokio::time::sleep(Duration::from_secs(1)).await;
-
-    // Create a factory to remove the warning about unused new() function.
-    let _factory = ReplicatorConsumerFactory::new(
-        Arc::new(
-            SessionBuilder::new()
-                .known_node("127.0.0.1:9042")
-                .build()
-                .await?,
-        ),
-        "ks".to_string(),
-        "t".to_string(),
-    );
-
-    println!("Replicator");
-    Ok(())
+async fn main() -> anyhow::Result<()> {
+    let args = Args::parse();
+    let source_tables = args.table.split(',').map(|s| s.to_string()).collect();
+
+    let (mut replicator, handle) = Replicator::new(
+        args.source,
+        args.keyspace.clone(),
+        source_tables,
+        args.destination,
+        args.keyspace,
+        args.table,
```

Comment:
This doesn't look correct. You adjusted the CDC log reader to be able to read from multiple tables at the same time, but the replicator consumer still supports only one output table. Besides, you are giving it a comma-separated string of table names as the output table, which is guaranteed not to work when more than source table is specified.

Please adjust the replicator so that it supports multiple tables + add a test for it. If you wish, we can postpone it to a followup (but, in such case, please remove the ability to specify more than one source table for the replicator).